lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/remote/RemoteConfiguration.java

```diff
@@ -98,6 +101,33 @@
   }
 
   /**
+   * The local address provider.
```

Comment:
For these interfaces, I don't believe we need ``NamedParameter`` if their default class is not different from the annotation in their respective ``.java`` files.